sqlflow/client.py

```diff
@@ -0,0 +1,69 @@
+import os
+import logging
+import grpc
+import google.protobuf.wrappers_pb2 as wrapper
+
+import sqlflow.proto.sqlflow_pb2
+import sqlflow.proto.sqlflow_pb2_grpc
+
+_LOGGER = logging.getLogger(__name__)
+
+class Client(object):
+    def __init__(self, server_url=None):
+        """A minimum client that issues queries to and fetch results/logs from sqlflowserver.
+
+        Args:
+            server_url(str):
+                sqlflowserver url
+
+        Raises:
+            KeyError:
+                Raised if SQLFLOWSERVER is not specified as environment variable
+        """
+        if server_url is None:
+            try:
+                server_url = os.environ["SQLFLOW_SERVER"]
+            except KeyError:
+                _LOGGER.error("Please set system variable SQLFLOW_SERVER")
+                raise
+
+        channel = grpc.secure_channel(server_url)
+        self._stub = sqlflow_pb2_grpc.SQLFlowStub(channel)
+
+    def execute(self, operation):
```

Comment:
supports multiple sql with variable arguments? *operations?

Reply:
I would prefer `execute` takes one sql at a time, and write for loop outside
```python
for sql in sql_list:
    execute(sql)
```